mmdet3d/core/bbox/structures/utils.py

```diff
@@ -122,7 +122,17 @@ def points_cam2img(points_3d, proj_mat):
         torch.Tensor: Points in image coordinates with shape [N, 2].
     """
     points_num = list(points_3d.shape)[:-1]
+
     points_shape = np.concatenate([points_num, [1]], axis=0).tolist()
+    d1, d2 = proj_mat.shape[:2]
+    assert (d1 == 3 and d2 == 3) or (d1 == 3 and d2 == 4) or (d1 == 4
+                                                              and d2 == 4)
```

Comment:
Please add instruction to the assertion like assert ..., "The shape of projection matrix is not supported."